price-server/src/provider/fiat/quoter/CurrencyLayer.ts

```diff
@@ -14,17 +14,18 @@ interface Response {
 export class CurrencyLayer extends Quoter {
   private async updatePrices(): Promise<void> {
     const params = {
-      access_key: this.options.apiKey,
+      // access_key: this.options.apiKey,
       source: 'USD',
       currencies: this.symbols
         .map((symbol) => (symbol === 'USD/SDR' ? 'XDR' : symbol.replace('USD/', '')))
         .join(','),
     }
 
     const response: Response = await fetch(
-      `https://apilayer.net/api/live?${toQueryString(params)}`,
+      `https://api.apilayer.com/currency_data/live?${toQueryString(params)}`,
```

Comment:
are you sure?

https://apilayer.net/api/live?access_key=XXX&currencies=EUR,GBP,CAD,PLN&source=USD&format=1

works

 curl -i 'https://api.apilayer.com/currency_data/live?access_key=XXXX&currencies=EUR,GBP,CAD,PLN&source=USD&
format=1'

gives a 401 error

Reply:
i was able to get it working with this PR with our newly registered currencylayer & fixer apis, but not with 'https://apilayer.net/api/live?access_key=XXX&currencies=EUR,GBP,CAD,PLN&source=USD&format=1' 

seems like newer plans that are purchased via apilayer are only working with the new endpoint while old ones don't? that would be a real api mess

Reply:
What about `https://api.currencylayer.com/live?access_key=xxx&currencies=AUD,EUR,GBP,PLN&source=USD&`? This works for my old API key, and is reflected in the [docs](https://currencylayer.com/documentation)

Reply:
> are you sure?
> 
> https://apilayer.net/api/live?access_key=XXX&currencies=EUR,GBP,CAD,PLN&source=USD&format=1
> 
> works
> 
> curl -i 'https://api.apilayer.com/currency_data/live?access_key=XXXX&currencies=EUR,GBP,CAD,PLN&source=USD& format=1'
> 
> gives a 401 error

from their docs you should use 

```
curl --request GET 'https://api.apilayer.com/currency_data/live?base=USD&symbols=EUR,GBP' \
--header 'apikey: YOUR API KEY'
```

old URLs don't work with newly registered accounts